src/main/java/com/superzanti/serversync/ServerSync.java

```diff
@@ -105,7 +105,6 @@ private void commonInit() {
 
     private void runInServerMode() {
         ServerSync.MODE = EServerMode.SERVER;
-        new Logger("server");
```

Comment:
this pseudo-initialization is no longer needed, as Logger initializes lazily at the first call using `ServerSync.MODE` as context identifier